system-test/src/systemTest/java/org/apache/mesos/elasticsearch/systemtest/ScalingSystemTest.java

```diff
@@ -81,6 +84,25 @@ public void shouldNotLoseDataWhenScalingDown() throws UnirestException {
         esTasks.waitForCorrectDocumentCount(DataPusherContainer.CORRECT_NUM_DOCS);
     }
 
+    @Test
+    public void shouldNoHaveStaleData() throws UnirestException, IOException {
+        LOGGER.info("Scaling down to 2 nodes");
+        scaleNumNodesTo(ipAddress, 2);
+        esTasks.waitForGreen();
+        esTasks.waitForCorrectDocumentCount(DataPusherContainer.CORRECT_NUM_DOCS);
+        List<String> esAddresses = esTasks.getEsHttpAddressList();
+
+        JsonNode body = Unirest.delete("http://" + esAddresses.get(0) + "/shakespeare-2016.02").asJson().getBody();
```

Comment:
This path will change with the date: "2016.02". This will only work in Feburary 2016. Use all instead: 
```
"http://" + esAddresses.get(0) + "/_all"
```